projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/jetpack.js

```diff
@@ -12,7 +12,7 @@ export default class JetpackPage extends Page {
 
 	async connect() {
 		const connectButtonSelector = '.jp-connect-full__button-container .dops-button';
-		return await page.click( connectButtonSelector );
+		return await page.click( connectButtonSelector, { timeout: 60000 } );
```

Comment:
Not really sure, but I think Jest times out before this. I think Jest has a 30s timeout and we shouldn't have PW timeout bigger than that.

Reply:
I'm not sure about it. we have few more calls like this and I think it works as expected